modDbImpl/src/org/aion/db/impl/rocksdb/RocksDBWrapper.java

```diff
@@ -59,25 +64,41 @@ private Options setupRocksDbOptions() {
         Options options = new Options();
 
         options.setCreateIfMissing(true);
+        options.setUseFsync(false);
         options.setCompressionType(
                 enableDbCompression
-                        ? CompressionType.SNAPPY_COMPRESSION
+                        ? CompressionType.LZ4_COMPRESSION
                         : CompressionType.NO_COMPRESSION);
 
+        options.setBottommostCompressionType(CompressionType.LZ4_COMPRESSION);
+        options.setMinWriteBufferNumberToMerge(4);
+        options.setLevel0StopWritesTrigger(512);
+        options.setLevel0SlowdownWritesTrigger(0);
+
         options.setWriteBufferSize(this.writeBufferSize);
         options.setRandomAccessMaxBufferSize(this.readBufferSize);
         options.setParanoidChecks(true);
         options.setMaxOpenFiles(this.maxOpenFiles);
         options.setTableFormatConfig(setupBlockBasedTableConfig());
+        options.setDisableAutoCompactions(false);
+        options.setIncreaseParallelism(max(1, Runtime.getRuntime().availableProcessors() / 2));
+
+        options.setLevelCompactionDynamicLevelBytes(true);
+        options.setMaxBackgroundCompactions(4);
+        options.setMaxBackgroundFlushes(2);
+        options.setBytesPerSync(1048576);
+        options.setCompactionPriority(CompactionPriority.MinOverlappingRatio);
+        options.optimizeLevelStyleCompaction(128 * 1024 * 1024);
```

Comment:
please add a constant for this in `RocksDBConstants`, same comment for the values above